server/subscribe.go

```diff
@@ -148,6 +149,10 @@ func (p *Plugin) matchesSubsciptionFilters(wh *webhook, filters SubscriptionFilt
 		return false
 	}
 
+	if filters.IssueStatuses.Len() != 0 && !filters.IssueStatuses.ContainsAny(issue.Fields.Status.ID) {
+		return false
+	}
```

Comment:
We also need to check for exclude right?

Also, are we able to put this into the `SubscriptionFilters.Fields` instead of making a separate `SubscriptionFilters.IssueStatuses`? This way we can reuse some logic for comparing field values below